### PR TITLE
Force TypeScript to transpile files one at a time

### PR DESCRIPTION
**Pull request by mprobst**
Cuts Angular's transpilation time from ~80 secs to 2.4 secs on my machine. x10 moon shots, baby!

### CODE REVIEW

**Comment by mprobst** [comment]
@alexeagle PTAL, this should be good to go in.

I'd need to mock "fs" to properly test this, or create a /tmp filesystem, but the code is trivial enough to not warrant that, I think.

File: lib/main.ts [hunk 1]
```diff
@@ -33,15 +33,39 @@ export class Transpiler {
         .join('\n');
   }
 
+  createCompilerHost(files: string[]): ts.CompilerHost {
+    var fileMap: {[s: string]: boolean} = {};
```
Comment (alexeagle) [inline lib/main.ts]: isn't the object key always a string?

**Comment by alexeagle** [comment]
LGTM

File: lib/main.ts (hunk 1 above)
Comment (mprobst) [inline lib/main.ts]: Yes, but TS wants you to spell it out, for some reason.

**Pull request merged**